docs/www/mirrormaker-migration.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Migrating data to Redpanda
+order: 2
+---
+
+# Migrating data to Redpanda
+
+One of the more elegant aspects of Redpanda is that it is compatible with the [Kafka API and ecosystem](/docs/www/faq.md).
+So, when you want to migrate data from Kafka or replicate data between Redpanda clusters,
+MirrorMaker 2 tool, bundled in the [Kafka download package](https://kafka.apache.org/downloads), is a natural solution.
+
+In this article, we'll go through the process of configuring MirrorMaker to replicate from one Redpanda cluster to another.
+For all of the details on MirrorMaker and its options, check out the [Kafka documentation](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=27846330).
+
+## Prerequisites
+
+To set up the replication, we'll need:
+
+- 2 Redpanda clusters - You can set up these clusters in any deployment you like, including [Kubernetes](/docs/www/quick-start-kubernetes.md), [Docker](/docs/www/quick-start-docker.md), [Linux](/docs/www/quick-start-linux.md), or [MacOS](/docs/www/quick-start-macos.md).
+    The key points here are:
+    - Make sure that the IP addresses and ports on each cluster are accessible by the  MirrorMaker.
+    - Create topics in the destination cluster that match the names of the topics that you want to replicate from the source cluster.
+
+- An instance of MirrorMaker - You can set up a separate system with MirrorMaker or run MirrorMaker on one of the Redpanda clusters.
+
+## Installing MirrorMaker
```

Comment:
we should fix this and it should be in our packaged bins. 

@ivotron - can we add this and add that to the license files for `mm2`

Reply:
we should release this when someone doesn't have to download mm2 from the apache mirror.

Reply:
sounds good. I've added it in our internal ticket tracker

Reply:
I'll update this for MM2 but won't release it until the script is in our bins.

Reply:
@ivotron we should have this in `vtools` production next by next tuesday 5/25